src/VisualStudio/VisualBasic/Impl/BasicVSResources.Designer.vb

```diff
@@ -110,6 +110,15 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
         End Property
         
         '''<summary>
+        '''  Looks up a localized string similar to _Always add new line on enter.
+        '''</summary>
+        Friend Shared ReadOnly Property Option_AlwaysInsertNewlineOnEnter() As String
+            Get
+                Return ResourceManager.GetString("Option_AlwaysInsertNewlineOnEnter", resourceCulture)
```

Comment:
my preference is to have tehse exactly match the english string, just with underscores.  So if the english string is "Always insert newline on enter" then the resource name should be Option_Always_insert_newline_on_enter"